ios/ReactTestApp/Manifest.swift

```diff
@@ -7,21 +7,25 @@
 
 import Foundation
 
-struct Component: Decodable {
+struct Component: Codable {
     let displayName: String?
     let initialProperties: [String: String]?
 }
 
-struct Manifest: Decodable {
+struct Manifest: Codable {
     let name: String
     let displayName: String
     let components: [String: Component]
 
-    static func readFile() -> Manifest? {
+    static func fromFile() -> Manifest? {
```

Comment:
Would it make sense to parameterise this function, or to provide an overloaded method that accepts a path / json? I am thinking of scenarios where manifest might be not in the project root, or when the user might have several manifests?

Reply:
Let's add overloads when we'll actually need them.